db/index.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 
-mongoose.connect('mongodb://localhost/mykea', { useNewUrlParser: true })
+mongoose.connect('mongodb://localhost/mykea', { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false })
```

Comment:
I'm not super familiar with mongo but what does 'useUnifiedTopology: true, useFindAndModify: false' do?

Reply:
@javsflores These basically are just to fix deprecation warnings! They aren't really altering functionality as far as I know -- I just read the deprecation warnings and added the statements the warnings said would fix them. I don't think they're super important. My understanding from fixing it that useFindAndModify() used to be what findOneAndUpdate() would delegate to, but is now separate. And I believe that useUnifiedTopology relates to how Mongo handles connections!